app/models/tag.rb

```diff
@@ -1,5 +1,7 @@
 class Tag < ActiveRecord::Base
 	has_and_belongs_to_many :bruse_files
+  attr_accessor :file_id
```

Comment:
These `attr_accessor` in this file and in the _BruseFile_ model is apparently unnecessary. I tried without them and it seems to work fine.